edgraph/access_ee_test.go

```diff
@@ -0,0 +1,112 @@
+package edgraph
```

Comment:
add the copyright, this will be under community license (not apache v2) and also the right tags.